src/Microsoft.ML.StandardLearners/Standard/Online/OnlineGradientDescent.cs

```diff
@@ -95,22 +95,22 @@ public override LinearRegressionModelParameters CreatePredictor()
         /// <param name="featureColumn">Name of the feature column.</param>
         /// <param name="learningRate">The learning Rate.</param>
         /// <param name="decreaseLearningRate">Decrease learning rate as iterations progress.</param>
-        /// <param name="l2RegularizerWeight">L2 Regularization Weight.</param>
+        /// <param name="l2Regularization">Weight of L2 regularization term.</param>
         /// <param name="numIterations">Number of training iterations through the data.</param>
         /// <param name="lossFunction">The custom loss functions. Defaults to <see cref="SquaredLoss"/> if not provided.</param>
         internal OnlineGradientDescentTrainer(IHostEnvironment env,
```

Comment:
```suggestion
        internal OnlineGradientTrainer(IHostEnvironment env,
```
A descent algorithm ensures the decrease of function value per iteration. However, this is not true for most stochastic gradient learners.

Reply:
I don't get your comment at all.

---
In reply to: [263972819](https://github.com/dotnet/machinelearning/pull/2892#discussion_r263972819) [](ancestors = 263972819)